src/devices/uFire_ISE/UFireIse.cs

```diff
@@ -0,0 +1,388 @@
+﻿// Licensed to the.NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device;
+using System.Device.I2c;
+using UnitsNet;
+
+namespace Iot.Device.UFire
+{
+    /// <summary>
+    /// μFire ISE (Ion Specific Electrode) Probe Interface controller
+    /// </summary>
+    public class UFireIse : IDisposable
+    {
+        private const int ISE_TEMP_MEASURE_TIME = 750;
+        private const int ISE_MV_MEASURE_TIME = 250;
+        private const int ISE_COMMUNICATION_DELAY = 10;
+
+        private const bool ISE_DUALPOINT_CONFIG_BIT = false;
+        private const int ISE_TEMP_COMPENSATION_CONFIG_BIT = 1;
+
+        private I2cDevice _device;
+
+        private float _mV = 0;
+        private Temperature _temp = new Temperature();
+
+        private bool _temperatureCompensation = false;
+
+        /// <summary>
+        /// Use temperature compensation
+        /// </summary>
+        public bool TemperatureCompensation
+        {
+            get
+            {
+                return _temperatureCompensation;
+            }
+
+            set
+            {
+                _temperatureCompensation = value;
+
+                byte retval = Read_byte((byte)Register.ISE_CONFIG_REGISTER);
+
+                retval = (byte)Bit_set((int)retval, ISE_TEMP_COMPENSATION_CONFIG_BIT, _temperatureCompensation);
+
+                Write_byte((byte)Register.ISE_CONFIG_REGISTER, retval);
+            }
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="UFireIse"/> class.
+        /// </summary>
+        /// <param name="i2cDevice">The I2C device to be used</param>
+        public UFireIse(I2cDevice i2cDevice)
+        {
+            _device = i2cDevice;
+        }
+
+        /// <summary>
+        /// Measure mV
+        /// </summary>
+        /// <returns>mV read from ISE Probe Interface</returns>
+        public ElectricPotential MeasuremV()
```

Comment:
Now you have a UnitsNet, the name should be changed to remove the unit. So ```Measure``` or ```ProbePotential``` or anything like this
Also, this is valid for all variables and properties like this in all the PR.
I understand that because all probes are different, the range will be different. That said, for the comment, would be good to explain if there is any typical range. And adjust the comment to say that the best way to use the measurement is in millivolt. So adjust comment with something like:
/// Measure the ISE Probe Interface, typical measure are in the millivolt range